Cargo.toml

```
@@ -39,3 +40,15 @@ tracing = "0.1"
 tracing-subscriber = "0.3"
 tree_hash = "0.9.0"
 tree_hash_derive = "0.9.0"
+discv5 = "0.9.0"
+tokio = { version = "1.42.0", features = ["rt", "rt-multi-thread", "sync", "signal", "time","macros"] }
+log = "0.4.2"
+futures = "0.3.30"
+#tiny-keccak = { version = "2.0.2", features = ["keccak"] }
+hex = "0.4.3"
+async-channel = "2.3.1"
+
+# local dependencies
+task_executor = { path = "crates/common/task_executor" }
```

```suggestion
task-executor = { path = "crates/common/task_executor" }
```

can we use - to be consistent

---

name it ream-task-executor as well please

---

I was just thinking that.  coolios!